test/metabase/util/password_test.clj

```diff
@@ -47,12 +49,29 @@
           (is (= expected
                  (apply #'pwu/password-has-char-counts? input))))))))
 
-(deftest is-complex?-test
+(deftest is-valid?-normal-test
   (testing "Do some tests with the default (:normal) password requirements"
-    (doseq [[input expected] {"ABC"    false
-                              "ABCDEF" false
-                              "ABCDE1" true
-                              "123456" true}]
-      (testing (pr-str (list 'is-complex? input))
+    (doseq [[input expected] {"ABC"              false
+                              "ABCDEF"           false
+                              "ABCDE1"           false
+                              "123456"           false
+                              "passw0rd"         false
+                              "PASSW0RD"         false
+                              "unc0mmonpw"       true
+                              "pa$$w0®∂"         true
+                              "s6n!8z-6.gcJe"    true}]
```

Comment:
this is a really minor nitpick but you shouldn't have all those extra spaces between the map keys and map args. You'd should either align the map like

```clj
{"ABC"           false
 "ABCDEF"        false
 "ABCDE1"        false
 "123456"        false
 "passw0rd"      false
 "PASSW0RD"      false
 "unc0mmonpw"    true
 "pa$$w0®∂"      true
 "s6n!8z-6.gcJe" true}
```

or

```clj
{"ABC" false
 "ABCDEF" false
 "ABCDE1" false
 "123456" false
 "passw0rd" false
 "PASSW0RD" false
 "unc0mmonpw" true
 "pa$$w0®∂" true
 "s6n!8z-6.gcJe" true}
```

Either is acceptable, and you'll see both out in the wild; I usually do the former but only because CIDER does it automatically -- not sure I'd bother if I had to do it by hand.